R/utils.R

```diff
@@ -1,3 +1,314 @@
+#' Assign an additional name to an entity
+#'
+#' * `to_alias()` takes any character vector and creates an alias by
+#' transforming the input (a) to lower case; (b) to latin-ascii characters; and
+#' (c) to standard abbreviations of ownership types. Commonly, the inputs are
+#' values from the columns `name_direct_loantaker` or `name_ultimate_parent`
+#' of a loanbook dataset, or from the column `name_company` of an asset-level
+#' dataset.
+#' * `from_name_to_alias()` outputs a table giving default strings used to
+#' convert from a name to its alias. You may amend this table and pass it to
+#' `to_alias()` via the `from_to` argument.
+#'
+#' @section Assigning aliases:
+#' The transformation process used to compare names between loanbook and tilt
+#' datasets applies best practices commonly used in name matching algorithms:
+#' * Remove special characters.
+#' * Replace language specific characters.
+#' * Abbreviate certain names to reduce their importance in the matching.
+#' * Spell out numbers to increase their importance.
+#'
+#' @author person(given = "Evgeny", family = "Petrovsky", role = c("aut",
```

Comment:
For the accreditations, the author of the function is named here, but I will add the source in the description too ! :) @MirijaHa